.github/workflows/ci.yaml

```diff
@@ -461,30 +461,29 @@ jobs:
         uses: github/codeql-action/upload-sarif@v1
         with:
           sarif_file: "trivy-image-results.sarif"
-
+  # NOTE@jsjoeio 5/10/2021
+  # Disabling until fixed upstream
+  # See: https://github.com/aquasecurity/trivy-action/issues/22#issuecomment-833768084
   # We have to use two trivy jobs
   # because GitHub only allows
   # codeql/upload-sarif action per job
-  trivy-scan-repo:
-    runs-on: ubuntu-20.04
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Run Trivy vulnerability scanner in repo mode
-        # Commit SHA for v0.0.14
-        uses: aquasecurity/trivy-action@341f810bd602419f966a081da3f4debedc3e5c8e
-        with:
-          scan-type: "fs"
-          scan-ref: "."
-          ignore-unfixed: true
-          format: "template"
-          template: "@/contrib/sarif.tpl"
-          output: "trivy-repo-results.sarif"
-          severity: "HIGH,CRITICAL"
-
-      - name: Upload Trivy scan results to GitHub Security tab
-        uses: github/codeql-action/upload-sarif@v1
-        with:
-          sarif_file: "trivy-repo-results.sarif"
+  # trivy-scan-repo:
```

Comment:
if this was in its own workflow file, then you can disable the whole workflow in the UI :stuck_out_tongue_winking_eye: https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow

another way you can disable it that doesn't require commenting it all out, is to use `if: 1 == 2` or something in the job definition. We do that sometimes too https://github.com/cdr/m/blob/dba70b6a0c904e360dac0ed930028a0107ef2140/.github/workflows/nightly.yaml#L14

Reply:
> if this was in its own workflow file, then you can disable the whole workflow in the UI :stuck_out_tongue_winking_eye: https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow
> 
> another way you can disable it that doesn't require commenting it all out, is to use `if: 1 == 2` or something in the job definition. We do that sometimes too https://github.com/cdr/m/blob/dba70b6a0c904e360dac0ed930028a0107ef2140/.github/workflows/nightly.yaml#L14